spec/index.bs

```diff
@@ -4695,10 +4701,15 @@ Issue: convert "Valid Texture Copy Range" into an algorithm with parameters, sim
                 - [$validating GPUBufferCopyView$](|source|) returns `true`.
                 - |source|.{{GPUBufferCopyView/buffer}}.{{GPUBuffer/[[usage]]}} contains
                     {{GPUBufferUsage/COPY_SRC}}.
-                - [$validating GPUTextureCopyView$](|destination|) returns `true`.
+                - [$validating GPUTextureCopyView$](|destination|, |copySize|) returns `true`.
```

Comment:
Are these `copySize` additions from another cset?

Reply:
I could move this into a different PR. The existing validation only checked the whole-subresource-copied restriction for depth-stencil and multisampled textures on T2T copies, but in D3D12 the restriction is on T2T, T2B, and B2T.

Reply:
For multisampled this was because T2B and B2T can't do multisampled at all. But I think it's okay to be slightly redundant.